animations/index.md

```diff
@@ -1,159 +1,88 @@
 ---
 layout: page
-title: Animations in Flutter
+title: Flutter animations
 permalink: /animations/
 ---
 
 * TOC Placeholder
 {:toc}
 
-Well-designed animations makes a UI feel more intuitive,
-contribute to the slick look and feel of a polished app,
-and improve the user experience. Flutter's animation support
-makes it easy to implement a variety of animation types.
-Many widgets, especially
-[Material Design widgets](https://flutter.io/widgets/material/),
-come with the standard motion effects defined in their design spec,
-but it's also possible to customize these effects.
-
-The following resources are a good place to start learning the Flutter
-animation framework. Each of these documents shows, step by step, how
-to write animation code.
-{% comment %}
-More documentation is in the works on how to implement common design
-patterns, such as shared element transitions,
-and physics-based animations.
```

Comment:
I'd like to see physics-based animations mentioned somewhere in this doc. Even though we might not have additional content for it for a while, we need to make sure people know it's possible to create physics-based animations. This is especially important if the developer wants to mimic certain iOS-specific animations.